ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-ref.yaml

```diff
@@ -13,8 +13,22 @@ ref:
   - namespace: test-credentials
     name: nutanix-pc
     mount_path: /var/run/vault/nutanix
+  env:
+  - name: SIZE_VARIANT
+    default: ""
+    documentation: |-
+      The size of the cluster in one of our supported t-shirt values that is standard across all CI environments.
+
+      The sizes are:
+      * "" (default) - 4 vCPU, 16GB control plane nodes, default workers
+      * "compact" - 8 vCPU, 32GB control plane nodes, no workers
+      * "large" - 16 vCPU, 64GB+ control plane nodes, default workers, suitable for clusters up to 250 nodes
```

Comment:
If no `large` and `xlarge` variables defined in the ipi-conf-nutanix-commands.sh, I'll prefer to remove them in the documentation.